### PR TITLE
Change toolbar title

### PR DESCRIPTION
**Pull request by u-nation**
## Issue
- #227

## Overview (Required)
- Please look at the gif :bow:

## Links
- None

## Screenshot
Before | After
:--: | :--:
 | 
| |

### CODE REVIEW

**Comment by konifar** [comment]
:eyes:

File: app/src/main/java/io/github/droidkaigi/confsched2017/view/activity/MainActivity.java [hunk 1]
```diff
@@ -46,21 +46,27 @@ protected void onCreate(Bundle savedInstanceState) {
     }
 
     private void initView() {
+        binding.toolbar.setTitle(getString(R.string.sessions));
+
         BottomNavigationViewHelper.disableShiftingMode(binding.bottomNav);
         binding.bottomNav.setOnNavigationItemSelectedListener(item -> {
             item.setChecked(true);
             switch (item.getItemId()) {
```
Comment (konifar) [inline app/src/main/java/io/github/droidkaigi/confsched2017/view/activity/MainActivity.java]: I think it's better.

```java
binding.toolbar.setTitle(item.getTitle());
```

**Comment by konifar** [comment]
Hmm, I guess it's better to add DroidKaigi white icon to the left side on Toolbar like Twitter app.

File: app/src/main/java/io/github/droidkaigi/confsched2017/view/activity/MainActivity.java (hunk 1 above)
Comment (u-nation) [inline app/src/main/java/io/github/droidkaigi/confsched2017/view/activity/MainActivity.java]: Oh, I agree with you.
I'll fix it!

**Comment by u-nation** [comment]
I'm sorry for taking long to write you back.
Let me try it!
Do you mean "DroidKaigi white icon" is ic_notification.png, don't you?

**Comment by u-nation** [comment]
How is this??
Is this what you wanted?? 😭 
## Screenshot
Before | After
:--: | :--:
||fix 227|

**Comment by konifar** [comment]
Sorry to be late 🙇 
Thanks for quick improvement. Can I request once more? I want you to change the icon to `ic_launcher.png` instead of `ic_notification.png` 🙏

**Comment by u-nation** [comment]
Of course! But I'm not sure which choice is better.　🙇

1.  no icon
2. ic_launcer(debug/release)
3. We will have a designer make it. 🙏 

Anyway, which do you wanted,  debug or release icon??(>_<)

debug | release
:--: | :--:
|device-2017-02-10-001011|device-2017-02-10-000848|

**Comment by konifar** [comment]
Thanks! Can you use white `ic_notification.png` icon?
I may change it but now it looks good 👍

**Comment by u-nation** [comment]
Of course!(>_<) Please let me reconfirm it.

Last PR already use white ic_notification.png. 🙇

I just renamed drawable-xxxhdpi/ic_notification.png to logo.png because I think the other size ic_notification.png is too small for toolbar.

I'm so sorry if I misunderstand your request. 🙇

**Comment by konifar** [comment]
Understood. Thanks! 👍

**Comment by u-nation** [comment]
I am sorry for taking up your time. 🙇
Thanks very much!(>_<)

**Pull request merged**